ui/components/multichain/pages/send/components/recipient-content.tsx

```diff
@@ -42,53 +39,37 @@ export const SendPageRecipientContent = ({
     asset.type !== AssetType.token &&
     asset.type !== AssetType.NFT;
 
-  const { asset: transactionAsset, amount } = useSelector(
-    getCurrentDraftTransaction,
-  );
+  const {
+    receiveAsset,
+    sendAsset,
+    amount: sendAmount,
+  } = useSelector(getCurrentDraftTransaction);
+
+  const bestQuote = useSelector(getBestQuote);
+
+  const amount =
+    receiveAsset.details?.address === sendAsset.details?.address
+      ? sendAmount
+      : { value: decimalToHex(bestQuote?.destinationAmount || '0') };
```

Comment:
Do fees need to be subtracted from `destinationAmount` or are they accounted for?

Reply:
Should be accounted for? Wouldn't all fees be on the source side